src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -508,6 +516,10 @@ after_credit( void *             _ctx,
 static void
 during_housekeeping( void * _ctx ) {
   fd_replay_tile_ctx_t * ctx = (fd_replay_tile_ctx_t *)_ctx;
+  FD_MCNT_SET( REPLAY, SNAPSHOT_STATUS_SNAPSHOT_BEGIN, 0 );
```

Comment:
I'm not quite sure what these counters are for, but this can't be right. Resetting them to zero will make prometheus think the application exited and restarted, counters should only increment.

Reply:
It made the visualization easier to understand IMO, I removed these lines since they're confusing